src/styles/blocks/search.css

```diff
@@ -0,0 +1,50 @@
+.search {
+  display: block;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  background-image: url('../../images/Search.svg');
+  background-repeat: no-repeat;
+  font-family: Avenir, Arial, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. It's a good practice to specify a generic font family as a fallback. For example, if 'Avenir' is not available, the browser can use 'Arial', and if 'Arial' is not available, it can fall back to any available sans-serif font.